package.json

```diff
@@ -21,9 +21,10 @@
     "test:integration:microblocks": "npm run devenv:deploy:pg -- -d && cross-env NODE_ENV=development jest --config ./jest.config.microblocks.js --coverage --no-cache --runInBand; npm run devenv:stop:pg",
     "test:integration:tokens": "npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.tokens.js --coverage --no-cache --runInBand; npm run devenv:stop",
     "git-info": "echo \"$(git rev-parse --abbrev-ref HEAD)\n$(git log -1 --pretty=format:%h)\n$(git describe --tags --abbrev=0)\" > ./.git-info",
-    "build": "npm run git-info && rimraf ./lib && tsc -p tsconfig.build.json",
+    "build": "npm run git-info && npm run build:api-documentation && rimraf ./lib && tsc -p tsconfig.build.json",
     "build:tests": "tsc -p tsconfig.json",
     "build:docs": "npm i --prefix docs && npm run build --prefix docs && npm i --prefix client && npm run generate:docs --prefix client",
+    "build:api-documentation": "npm i --prefix docs && npm run build --prefix docs",
```

Comment:
Any reason for this new command rather than using the existing `build:docs` one?